app/controllers/notes_controller.rb

```diff
@@ -1,10 +1,11 @@
 # Performs CRUD operation on notes table
 # It was generated with scaffold generator
 class NotesController < ApplicationController
-  before_action :set_note, only: [:show, :edit, :update, :destroy]
+  before_action :find_note, only: [:show, :edit, :update, :destroy]
```

Comment:
@sohilp393, use %i for the array of symbols.
change to `  before_action :find_note, only: %i[show edit update destroy]`

Reply:
@krishnasingh001.
Applied the changes as per the suggestions and the code is working fine.